openstack/blockstorage/v2/extensions/volumeactions/requests.go

```diff
@@ -172,3 +172,35 @@ func TerminateConnection(client *gophercloud.ServiceClient, id string, opts Term
 	})
 	return
 }
+
+// ExtendSizeOptsBuilder allows extensions to add additional parameters to the
+// ExtendSize request.
+type ExtendSizeOptsBuilder interface {
+	ToVolumeExtendSizeMap() (map[string]interface{}, error)
+}
+
+// ExtendSizeOpts contain options for extending the size of an existing Volume. This object is passed
+// to the volumes.ExtendSize function.
+type ExtendSizeOpts struct {
+	NewSize int `json:"new_size,omitempty"`
```

Comment:
>  Should I do something about it?

No, I was just curious if OpenStack would downsize as well. Nothing we can do without separately querying the current size.